packages/connect-examples/expo-playground/app/components/providers/README.md

```
@@ -0,0 +1,119 @@
+# SDK 初始化流程
+
+## 概述
+
+新的SDK初始化流程确保在应用启动时就正确设置transport类型，避免在各个组件中分散处理transport设置。
+
+## 初始化流程
+
+### 1. SDKProvider 启动
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)